frontend/src/pages/collaboration-service/CollaborationPage.tsx

```diff
@@ -115,21 +115,11 @@ export default function CollaborationPage() {
     )
   }
 
-  // When user ends session, remove user from collabStore
-  const endSession = async () => {
-    try {
-      await removeUserFromCollabStore(auth.id)
-      navigate("/matching/start")
-    } catch (error) {
-      console.error(error)
-    }
-  }
-
-  if (roomId == null) {
+  if (roomId === null || questionId === null) {
     return (
       <MainContainer className="px-4 text-center gap-3 flex flex-col">
         <h2 className="text-2xl">
-          You are at an invalid document ID for collaborating
+          Invalid collaboration arguments. Please try matching again.
```

Comment:
change this to `It seems like you are not in a valid collaboration environment. Please try matching again.` as well

Reply:
it's a duplicated `if` for checking null statements. I just deleted the whole deplicated `if` block